front/src/services/user.ts

```diff
@@ -19,3 +19,16 @@ export const updateNome = async (
     }
   );
 };
+export const updateSenha = async (
+  id: string,
+  dataS: { lastSenha: string; senha: string },
```

Comment:
```suggestion
  data: { lastSenha: string; senha: string },
```

acho que aqui o parâmetro pode ser só data, o que acha ?

Reply:
SIm, na versão que tenho tá data normal, foi minha dúvida de como chamar o dado lá no front. imaginei que precisava mudar pra nao conflitar com o findAllUsers mas não. vou ajustar isso.